### ISSUE TITLE
Tangle autoloads

### ISSUE THREAD

**Issue by ashfaqfarooqui** [alhassy/holy-books#7]
Hi,

Thanks for this package.

I am guessing the autoloads need to be tangled. As it is now, the minor modes are not defined.

**Comment by alhassy** [comment]
Would you mind please elaborating?

I'm glad you like the package; thanks for contributing!

**Comment by ashfaqfarooqui** [comment]
The following appears in the org file but not in the .el file.

```;;;###autoload
(define-minor-mode holy-books-mode
    "Org-mode links, tooltips, and Lisp look-ups for the Quran & Bible."
  nil nil nil)```

If I understand elisp correctly, the above piece of code is needed to tell emacs about the minor more. Or..? Am I missing something.

**Comment by alhassy** [comment]
Good catch! Fixed by commit 81e6b4b
